ta/pkcs11/src/pkcs11_token.c

```diff
@@ -189,6 +189,31 @@ static void pad_str(uint8_t *str, size_t size)
 	TEE_MemFill(str + n, ' ', size - n);
 }
 
+static void set_token_description(struct pkcs11_slot_info *info,
+				  unsigned int token_id)
+{
+	char desc[sizeof(info->slot_description) + 1] = { 0 };
+	TEE_UUID dev_id = { };
+	TEE_Result res = TEE_ERROR_GENERIC;
+	int n = 0;
+
+	res = TEE_GetPropertyAsUUID(TEE_PROPSET_TEE_IMPLEMENTATION,
+				    "gpd.tee.deviceID", &dev_id);
+	if (res == TEE_SUCCESS) {
+		n = snprintf(desc, sizeof(desc), PKCS11_SLOT_DESCRIPTION
```

Comment:
`snprintk()` is needed for `%pUl` to print a string representation of a UUID.

Reply:
ok. 
That's for consistency I guess? in optee both `snprintk` and `snprintf` end in the same `__vsnprintf`.

Reply:
Yes, but with a small difference, with `ext` passed as `true` or `false`.